components/home/SliderBar.vue

```diff
@@ -76,6 +76,13 @@
         </v-card>
       </v-img>
     </SwiperSlide>
+    <SwiperSlide class="parrentSlide">
+      <v-container class="linkContainer">
+        <nuxt-link to="/gatherings" class="showMore">
+          Посмотреть все <img src="@/assets/images/gatherings/arrow-point-to-right 1.svg" alt="go-to">
+        </nuxt-link>
+      </v-container>
+    </SwiperSlide>
```

Comment:
```suggestion
    <SwiperSlide class="my-auto text-center" style="width: 40%;">
      <v-btn text nuxt to="/gatherings">
        Посмотреть все <v-icon>mdi-chevron-right</v-icon>
      </v-btn>
    </SwiperSlide>
```

This is much simpler and doesn't need any additional CSS.